src/blog/2024/08/comparing-dashboard-2-with-uibuilder.md

```diff
@@ -0,0 +1,232 @@
+
+### Node-RED Dashboard 2.0
+
+When searching for `flowfuse/node-red-dashboard` on Google, the first result we get to the documentation, which is useful. However, in the Node-RED Palette Manager, finding the correct package can be confusing because there are many community third-party nodes and plugins that work with Node-RED Dashboard 2.0. This can make it bit confusing for new users to locate the right package.
+
+### UI-Builder
+
+Finding UI-Builder is straightforward, as its package name `node-red-ui-builder` is distinct and easily identifiable both on Google and in the Node-RED Palette Manager. This clear naming helps users quickly locate the correct package.
+
+## How Easy is it to Get Started?
+
+## Node-RED Dashboard 2.0
+
+Getting started with Node-RED Dashboard 2.0 is relatively easy due to its low-code approach. It features a sidebar for managing UI widgets, themes, configurations, and settings, with intuitive navigation to the dashboard page. This makes it accessible for users with varying levels of technical expertise.
+
+### UI-Builder
+
+UI-Builder can be more challenging to start with, as it does not follow a low-code approach and lacks a separate sidebar for managing UI elements. Navigating the dashboard built with UI-Builder can be less straightforward, requiring users to have a deeper understanding of coding and UI design principles.
+
+## How Easy is it to Migrate from Node-RED Dashboard ?
+
+[Node-RED Dashboard](https://flows.nodered.org/node/node-red-dashboard) is a module that provides a set of nodes in Node-RED to quickly create user interfaces or live data dashboards. It was developed by one of the creators of Node-RED and is the most used and downloaded package in the Node-RED ecosystem. However, it is now deprecated. For more information, refer to [Node-RED Dashboard Formally Deprecated](/blog/2024/06/dashboard-1-deprecated/).
+
+### Node-RED Dashboard 2.0
+
+Node-RED Dashboard 2.0 is developed to replace the the deprecated standard Node-RED Dashboard. It retains the core concepts and UI elements but introduces more advanced options and configurations. To facilitate the transition, FlowFuse, the creator of Node-RED Dashboard 2.0, provides a [migration service](/product/dashboard/#migration-service) that simplifies migration of flows or projects from the Node-RED Dashboard to Node-RED Dashboard 2.0. This service helps ensure a smoother migration process with minimal disruption.
+
+### UI-Builder
+
+Migrating from Node-RED Dashboard 1 to UI-Builder is significantly more complex. UI-Builder does not follow the same concepts or provide the same UI elements as the Node-RED Dashboard. Users will need to recreate their dashboards from scratch, as UI-Builder relies on custom coding and frontend frameworks rather than the predefined, low-code widgets of Node-RED Dashboard. This process can be overwhelming and requires a solid understanding of HTML, CSS and the frontend frameworks if you wanted use.
```

Comment:
```suggestion
Migrating from Node-RED Dashboard 1 to UI-Builder is significantly more complex. UI-Builder does not follow the same concepts or provide the same UI elements as the Node-RED Dashboard.

Users will need to recreate their dashboards from scratch, as UI-Builder relies on custom coding and frontend frameworks rather than the predefined, low-code widgets of Node-RED Dashboard. This process can be overwhelming and requires a solid understanding of HTML, CSS and the frontend frameworks if you wanted use.
```